### ISSUE TITLE
Input field readout is not the same as the selected date

### ISSUE THREAD

**Issue by onx2** [praveenpuglia/vuetify-daterange-picker#28]
For example, if I select september 6th - 11th in the date picker the input field reads 5th - 10th. This doesn't seem to be the case with the top readout inside of badges.

Is there a configuration that I am missing? This doesn't seem intuitive that the readout would be different than the dates selected. 

![image](https://user-images.githubusercontent.com/23039725/45439662-6d475900-b688-11e8-821f-e42f02281511.png)

**Comment by praveenpuglia** [comment]
That is weird. Can you post your options here?

**Comment by praveenpuglia** [comment]
1. The codesandbox link is too old. Can you try reproduce it in this demo? - http://vuetify-daterange.surge.sh. The code for that is in [here](../tree/master/demo/App.vue)

2. Even in this codesandbox link I couldn't see wrong dates. May I know which browser you are using and any other details that you think might be relevant?

![image](https://user-images.githubusercontent.com/313363/45529953-bd97f580-b805-11e8-9a24-dc13cc5bc69a.png)

**Comment by onx2** [comment]
Going to that page gives me the same results:

Browser: Chrome Version 69.0.3497.92 (Official Build) (64-bit)
OS: Windows 10 Home - Version 10.0.17134 Build 17134

![image](https://user-images.githubusercontent.com/23039725/45551253-46f5fa80-b7fb-11e8-9dd1-5f532d675796.png)

**Comment by praveenpuglia** [comment]
Thanks so much for using it and making me feel proud. 
I somehow think this is a timezone issue. Because I never seem to be able to reproduce it. Can you please tell me which TZ are you in? I can try replicating this by emulating may be?

**Comment by onx2** [comment]
No problem, I'm very glad to have a robust solution for date range picking. I'm also excited for the colorizing feature on the way between ranges.

Timezone: UTC -5:00 Eastern Time (US & Canada)

The timezone being an issue makes a lot of sense. Perhaps the plugin you are using to handle dates is using GMT by default and not snagging the user's timezone?

**Comment by praveenpuglia** [comment]
Yeah! may be that's the issue. I'll look at it at the earliest. Going through a bit complicated phase right now. but I'll get back to it soon.

**Comment by StewM** [comment]
It seems to be related to the format being used for the start date and end date. new Date('2018-10-01') returns Sun Sep 30 2018 19:00:00 GMT-0500 (Central Daylight Time), but new Date('2018/10/01') returns Mon Oct 01 2018 00:00:00 GMT-0500 (Central Daylight Time). 
Standardizing the format being passed into Date in the computed methods would probably fix this.

**Comment by StewM** [comment]
I've got a working solution, I'll submit a PR after work when I have a chance to write it up.

**Comment by StewM** [comment]
Submitted a PR: #36

**Comment by praveenpuglia** [comment]
Hello everyone, new releases are here and this has been fixed thanks to @StewM . Let me and him know if it all works for you :)

Also, if any of you are interested into feature development for this component, let me know. I can add you as a collaborator and we can work together to make this even cooler.
